projects/golang/go/1.15/patches/0005-go-1.15.15-eks-math-big-prevent-overflow-in-R.patch

```diff
@@ -9,7 +9,8 @@ Backported To: go-1.15.15-eks
 Backported On: Thu, 22 Sept 2022
 Backported By: budris@amazon.com
 Backported From: release-branch.go1.16
-Source Commit: https://github.com/golang/go/commit/07ee9e6445057e181fb3895df978748ffef30327
+Upstream Source Commit: https://github.com/golang/go/commit/07ee9e6445057e181fb3895df978748ffef30327
+EKS Patch Source Commit: https://github.com/danbudris/go/commit/f56e2b486b1cba8eae7326dac48da67c16ebba28
```

Comment:
Is this for the patches that required differences between upstream an your commit? Should we be doing this format on all of our patches that differ? Mostly wondering if I should do it [here](https://github.com/aws/eks-distro/pull/1254) for the Kubernetes backports, or how this extends to other patching we are doing.

Reply:
I decided to include it on all of them in this specific project in order to ensure we're tracking everything that's done and can easily backtrack our steps if required as we get further into the testing phase. I think it'd be a reasonable to at least note in the k8s patch the changes that had to be made between the source commit and the patch, so that we have a record of what modifications we required